tests/AbstractSnowflakeTest.php

```diff
@@ -45,6 +45,9 @@ public function setUp(): void
         $this->connection->query(
             sprintf('USE SCHEMA %s', $this->connection->quoteIdentifier($config['parameters']['db']['schema']))
         );
+        $this->connection->query(
+            'alter session set client_timestamp_type_mapping=\'timestamp_ntz\''
+        );
```

Comment:
proc?

Reply:
no, jeste nevim proc :-)

Reply:
because if i try create timestamp column:
`"timestamp" TIMESTAMP DEFAULT CURRENT_TIMESTAMP`

so Snowflake return SQL error:
`Expression type does not match column data type, expecting TIMESTAMP_NTZ(9) but got TIMESTAMP_LTZ(9) for column timestamp, SQL state 22000 in SQLPrepare`